### PR TITLE
Add CI job for PHP 8.4

### PR DESCRIPTION
**Pull request by greg0ire**
Related:
* https://github.com/doctrine/orm/issues/11624

### CODE REVIEW

**Review by SenseException — DISMISSED** [review]

**Comment by SenseException** [comment]
Nice, that it seems to work with PHP 8.4 out of the box.

File: tests/Tests/TestInit.php [hunk 1]
```diff
@@ -15,10 +15,7 @@
 use function file_exists;
 use function mkdir;
 
-use const E_ALL;
-use const E_STRICT;
-
-error_reporting(E_ALL | E_STRICT);
+error_reporting(-1);
```
Comment (derrabus) [inline tests/Tests/TestInit.php]:
```suggestion
error_reporting(E_ALL);
```

**Review by derrabus — APPROVED** [review]

**Comment by derrabus** [comment]
Shall we target 2.19?

**Review by derrabus — DISMISSED** [review]

**Comment by beberlei** [comment]
We need to release 2.20 with a more undeprecations in the next few days anyways, so 2.20 for me is fine.

**Pull request merged**